_pages/extension_sharing_strategies.md

```diff
@@ -0,0 +1,143 @@
+    * **“ndx-custom-recipe”** refers to a specific recipe for deploying a particular extension. This repo is also part of the “nwb-extensions” GitHub organization. This repo should be automatically generated by the CI of the staged-extensions repo
+    **“user/ndx-custom-recipe”** refers to the users fork of the ndx-custom-recipe repo. This repo lives in the users Git space (NOT in nwb-extensions)
+
+### 1.2 Summary
+
+NWB:N defines a set of <a href="{{ site.url }}{{ site.baseurl }}/versioning_guidelines">Versioning Guidelines</a> for versioning extension namespaces and  <a href="{{ site.url }}{{ site.baseurl }}/ndx_sharing_guideline">NDX Sharing Guidelines</a> that public NDX MUST follow and internal NDXs are highly RECOMMENDED to follow. In order to help ensure compliance with these rules NWB:N provides a number of tools, templates, and guidelines.
+
+## 2. Methods and tools for sharing extensions
+
+### 2.1 [nwb-extensions](https://github.com/nwb-extensions): Extensions GitHub Organization
+
+NDX should be shared via a dedicated [nwb-extensions](https://github.com/nwb-extensions) GitHub organization. The NDX catalog provides the ndx-template and staged-extensions extension template repositories (see below) as well as all public ndx-custom-recipe repositories. In this way, all public extensions are managed in one place and MUST follow the same basic internal organization. This also avoids cluttering up the regular [Neurodata Without Borders GitHub organization](https://github.com/NeurodataWithoutBorders). Finally, this strategy allows for labs/universities/independent-groups to maintain the ndx-template source code repository in their own GitHub organization. The ability to do so, can be important in particular when the developers of the extension are funded by their own grants and/or are applying for funding.
+
+#### 2.1.1 [ndx-template](https://github.com/nwb-extensions/ndx-template): Extension template
+
+The [ndx-template](https://github.com/nwb-extensions/ndx-template) provides a standard Git template repository with a standard setup for creating and sharing extensions and instructions for how to use the template. Using [Cookiecutter](https://cookiecutter.readthedocs.io/en/latest/), users can easily modify the template to create the basic setup for their ndx-template repository. See the [ndx-template](https://github.com/nwb-extensions/ndx-template) repository for details on the organization of template.
+
+#### 2.1.2 [staged-extensions](https://github.com/nwb-extensions/staged-extensions): Creating custom ndx recipes
+
+The [staged-extensions](https://github.com/nwb-extensions/staged-extensions) repository is a place to submit NWB:N extension recipes before they become fully fledged NWB extensions.
```

Comment:
It's not clear what "fully fledged" means here. Proper public NDX's, I think?

Reply:
Yes, fully-fledged means before the ``ndx-custom-recipe`` repo exists in the catalog. Users create pull requests for new extensions on ``staged-extensions`` which then (when we merge the PR) automatically creates the ``ndx-custom-recipe`` and the extension becomes officially a "public NDX". After that the users will submit PRs to their ``ndx-custom-recipe`` directly.

Reply:
I updated the text accordingly to remove the term "fully-fledged" and instead describe that this is before there is a recipe for the extensions.